doc/tutorial/first_steps.md

```diff
@@ -1,380 +1,706 @@
+The MicroCloud setup process installs and initializes LXD. Thus, you can skip those sections on this page.
 ```
 
 ````
 
-This tutorial guides you through the first steps with LXD.
-It covers installing and initializing LXD, creating and configuring some instances, interacting with the instances, and creating snapshots.
+(tutorial-requirements)=
+## Requirements
 
-After going through these steps, you will have a general idea of how to use LXD, and you can start exploring more advanced use cases!
+- At least 20 GiB free disk space
+- A Linux distribution installed
 
-```{note}
-Ensure that you have 20 GiB free disk space before starting this tutorial.
-```
+(tutorial-install)=
+## Install LXD using snap
+
+This section of the tutorial assumes that you have the `snap` packaging system available on your system, which is the recommended way to use LXD.
 
-<!-- Include start tutorial installation -->
+If you use a Linux distribution that does not support `snap`, see {ref}`installing-other` and skip to the next section of this tutorial.
 
-## Install and initialize LXD
+(tutorial-install-requirements)=
+### Snap installation requirements
 
-The easiest way to install LXD is to install the snap package.
-If you prefer a different installation method, or use a Linux distribution that is not supported by the snap package, see {ref}`installing`.
+- The LXD snap must be [available for your Linux distribution](https://snapcraft.io/lxd#distros).
+- The [`snapd` daemon](https://snapcraft.io/docs/installing-snapd) must be installed.
 
-1. Install `snapd`:
+The `snapd` daemon that manages snap packages comes pre-installed on many distributions by default. To confirm whether it is available on your system, run:
 
-   1. Run `snap version` to find out if snap is installed on your system:
+```bash
+snap version
+```
+
+If you see an error message indicating that `snap` is not installed, visit the [Snap installation documentation](https://snapcraft.io/docs/installing-snapd) and follow the instructions there to install it.
 
-      ```{terminal}
-      :input: snap version
+Once you have confirmed that `snap` is available on your system, use it to install LXD:
 
-      snap    2.63+24.04ubuntu0.1
-      snapd   2.63+24.04ubuntu0.1
-      series  16
-      ubuntu  24.04
-      kernel  5.15.0-117-generic
-      ```
+```bash
+sudo snap install lxd
+```
 
-      If you see a table of version numbers, snap is installed and you can continue with the next step of installing LXD.
+If you see an error message that the LXD snap is already installed, run the following command to refresh it and ensure that you are running an up-to-date version:
 
-   1. If the command returns an error, run the following commands to install the latest version of ``snapd`` on Ubuntu:
+```bash
+sudo snap refresh lxd
+```
 
-          sudo apt update
-          sudo apt install snapd
+(tutorial-adduser)=
+## Add the current user to the `lxd` group
```

Comment:
We should probably mention (or link to the existing statement on this matter) about the ramifications of adding the current user to the `lxd` user, in that it gives them effectively root access.